cmd/lotus-miner/sectors.go

```diff
@@ -584,32 +583,6 @@ var sectorsListUpgradeBoundsCmd = &cli.Command{
 	},
 }
 
-var sectorsRefsCmd = &cli.Command{
-	Name:  "refs",
-	Usage: "List References to sectors",
-	Action: func(cctx *cli.Context) error {
-		nodeApi, closer, err := lcli.GetMarketsAPI(cctx)
```

Comment:
I don't think this section should be removed, it's not markets related but just happens to use `GetMarketsAPI` to get the `StorageMinerAPI`. Replace it with this and it should work:

```go
		nodeApi, closer, err := lcli.GetStorageMinerAPI(cctx)
```

Reply:
Done

Reply:
You are right, fixed this.